sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/UnconditionalJumpStatement.cs

```diff
@@ -0,0 +1,211 @@
+using System;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        public void Test1(string[] strings)

```

Comment:
Don't know what the rule should do if the `break` or `continue` is within an always true branch in a loop. Could you add a test?